src/tasks/runstart.ts

```diff
@@ -89,6 +90,14 @@ import { baseOutfit, unbreakableUmbrella } from "../outfit";
 import { excludedFamiliars } from "../resources";
 import { chooseFamiliar, cookbookbat, melodramedary, sombrero } from "../familiars";
 
+const bestStillsuitFamiliar = have($familiar`quantum entangler`)
+  ? $familiar`quantum entangler`
+  : have($familiar`Gelatinous Cubeling`)
+    ? $familiar`Gelatinous Cubeling`
+    : have($familiar`Blood-Faced Volleyball`)
+      ? $familiar`Blood-Faced Volleyball`
+      : $familiar`Mosquito`;
+
```

Comment:
This seems quite unwieldy to add on in the future. We can do something like
```
const bestStillsuitFamiliar = $familiars`quantum entangler, Gelatinous Cubling, Blood-Faced Volleyball, Mosquito`.filter(have)?.at(0) ?? $familiar.none;
```
or the way we do it in `chooseFamiliar()` in `familiar.ts`